src/lib/microservices/status-service/status-service.ts

```diff
@@ -0,0 +1,12 @@
+import * as statusManager from '../../common/status/status';
```

Comment:
I think this file is not necessary, I think `src/functions/azure/statusservice.ts` can use the `statusManager` directly.